models/items.js

```diff
@@ -0,0 +1,190 @@
+const firestore = require("../utils/firestore");
+
+const itemModel = firestore.collection("items");
+const tagModel = firestore.collection("tags");
+const levelModel = firestore.collection("levels");
+
+/**
+ *
+ * @param itemData { Object }: Data of the item, item type and Array of pairs of tagid and levelid
+ * @returns {Promise<tagModel|String>}
+ */
+
+const addTagsToItem = async (itemData) => {
+  try {
+    const { itemid, itemType, tagPayload } = itemData;
+    const batch = firestore.batch();
+    for (const tag of tagPayload) {
+      const tagData = await tagModel.doc(tag.tagid).get();
+      const levelData = await levelModel.doc(tag.levelid).get();
+      const itemTag = {
+        itemid,
+        itemtype: itemType.toUpperCase(),
+        tagid: tag.tagid,
+        tagname: tagData.data().name,
+        tagtype: tagData.data().type,
+        levelid: tag.levelid,
+        levelname: levelData.data().name,
+        levelnumber: levelData.data().levelnumber,
```

Comment:
⚠️ Why is there no distinction between two words?
This should be camelcased or separated by an underscore